sdv/metadata/metadata.py

```diff
@@ -66,9 +69,28 @@ def _set_metadata_dict(self, metadata, single_table_name=None):
             super()._set_metadata_dict(metadata)
         else:
             if single_table_name is None:
-                single_table_name = 'default_table_name'
+                single_table_name = self.DEFAULT_SINGLE_TABLE_NAME
             self.tables[single_table_name] = SingleTableMetadata.load_from_dict(metadata)
 
+    def _get_single_table_name(self):
+        """Get the table name if there is only one table.
+
+        Checks to see if the metadata contains only a single table, if so
+        return the name. Otherwise raise an error.
+
+        Args:
+            metadata (dict):
+                Python dictionary representing a ``MultiTableMetadata`` or
+                ``SingleTableMetadata`` object.
+        """
+        if len(self.tables) != 1:
+            warnings.warn(
+                'This metadata does not contain a single table. Could not determine '
```

Comment:
This reads as there are no tables. Maybe change to
```suggestion
                'This metadata contains more than one table. Could not determine '
```

Reply:
Updated to this:
`'This metadata does not contain only a single table. Could not determine '
                'single table name and will return None.'`
                
  Because it could also throw this warning with 0 tables